agent/04_agent_configure.sh

```diff
@@ -273,6 +284,79 @@ EOF
   fi
 }
 
+function generate_install_config_agent_config() {
+
+  MANIFESTS_PATH="${OCP_DIR}"
+  mkdir -p ${MANIFESTS_PATH}
+  
+  setNetworkingVars
+
+    cat > "${MANIFESTS_PATH}/agent-config.yaml" << EOF
+apiVersion: v1alpha1
+metadata:
+  name: ${CLUSTER_NAME}
+  namespace: ${CLUSTER_NAMESPACE}
+rendezvousIP: ${AGENT_NODES_IPS[0]}
+EOF
+    pull_secret=$(cat $PULL_SECRET_FILE)
```

Comment:
Here it's required a `set +x` to avoid leaking the pull secret in the standard output (even better at the beginning of the function)